modules/gapi/test/gapi_transform_tests.cpp

```diff
@@ -0,0 +1,99 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#include <tuple>
+
+#include "test_precomp.hpp"
+#include "opencv2/gapi/gtransform.hpp"
+
+namespace opencv_test
+{
+
+namespace
+{
+using GMat2 = std::tuple<GMat, GMat>;
+using GMat3 = std::tuple<GMat, GMat, GMat>;
+
+GAPI_TRANSFORM(my_transform, <cv::GMat(GMat2)>, "does nothing")
```

Comment:
Suppose all the specific information will be defined in parameters inside the class, and no non-G* types are passed in signature (otherwise there will be a compilation error)